tests/functional/processes/frbr/test_frbr_process.py

```diff
@@ -25,3 +26,53 @@ def test_frbr_process(db_manager, unfrbrized_record_uuid):
     classify_count = db_manager.session.query(Record).filter(Record.source == 'oclcClassify', Record.date_modified >= time_window).count()
 
     assert classify_count >= 1, f"Expected oclcClassify records. Found: {classify_count}"
+
+    new_record = (
+        db_manager.session.query(Record)
+        .filter(Record.title.ilike(f"%{unfrbrized_title}%"))
+        .order_by(Record.date_created.desc())
+        .first())
+    
+    if not new_record:
+        pytest.fail(f"No record found containing title: '{unfrbrized_title}'")
+
+    # Verify OWI in identifiers instead of source_id
+    assert any(id.endswith('|owi') for id in new_record.identifiers), \
+        f"Missing OWI identifier in: {new_record.identifiers}"
+
+    # Verify OCLC in identifiers
+    oclc_ids = [id.split('|')[0] for id in new_record.identifiers if id.endswith('|oclc')]
+    assert len(oclc_ids) > 0, f"No OCLC numbers found in: {new_record.identifiers}"
+
+    initial_catalog_count = db_manager.session.query(Record).filter(Record.source == 'oclcCatalog').count()
+
+    catalog_process = CatalogProcess(
+        'catalog_test',
+        None,
+        datetime.now(timezone.utc) - timedelta(minutes=2),
+        None
+    )
+
+    catalog_process.runProcess()
```

Comment:
This will wait for awhile to process more messages as they come into the queue. Let's set the max attempts to 1 so it only checks once. 

`catalog_process.runProcess(max_attempts=1)`